regulations/static/regulations/js/source/views/comment/comment-confirm-view.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var $ = require('jquery');
+var URI = require('urijs');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var MainEvents = require('../../events/main-events');
+var PreambleHeadView = require('../header/preamble-head-view');
+var CommentEvents = require('../../events/comment-events');
+var comments = require('../../collections/comment-collection');
+
+var CommentConfirmView = Backbone.View.extend({
+  initialize: function(options) {
+    Backbone.View.prototype.setElement.call(this, '#' + options.id);
+
+    this.findElms();
+
+    this.docId = this.$el.data('doc-id');
+    this.metaUrl = this.$el.data('meta-url');
+
+    this.listenTo(CommentEvents, 'read:proposal', this.handleRead);
+
+    if (this.metaUrl) {
+      this.poll(this.metaUrl);
+      comments.filter(this.docId).forEach(function(comment) {
+        comment.destroy();
+      });
+    }
+  },
+
+  findElms: function() {
+    this.$pdf = this.$el.find('.pdf');
+    this.$status = this.$el.find('.status');
+  },
+
+  handleRead: function() {
+    var section = this.docId + '-preamble-' + this.docId + '-I';
+    var options = {id: section, section: section, mode: 'read'};
+
+    $('#content-body').removeClass('comment-review-wrapper');
+
+    MainEvents.trigger('section:open', section, options, 'preamble-section');
+  },
```

Comment:
I don't disagree, but this is taken directly from `CommentReviewView`--@xtine may be able to say more about where it came from. We can have the template pass through the id of the first section in the toc instead--or just use a non-ajax link, since ajax navigation doesn't totally work with this view anyway.